lib/shopifyHmac/extractHmacParameter.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function extractHmacParameter({ query }) {
```

Comment:
as above I would give the method/file a shopify specific name (notwithstanding that shopify is also in the directory name). It might even be cleaner to have simply `lib/shopify/index.js`, and then export both methods. This would then make the developer experience like this:

```js
const {
  extractShopifySignature,
  constructShopifySignature
} = require('./lib/shopify')
```
```suggestion
function extractShopifySignature({ query }) {
```